src/Type/Symfony/InputInterfaceGetArgumentDynamicReturnTypeExtension.php

```diff
@@ -76,6 +77,11 @@ public function getTypeFromMethodCall(MethodReflection $methodReflection, Method
 			}
 		}
 
+		$method = $scope->getFunction();
+		if ($method instanceof MethodReflection && $method->getName() === 'interact') {
```

Comment:
You should also check the declaring class of the method. You might not be in a console command at all.

Reply:
I added a new check https://github.com/phpstan/phpstan-symfony/pull/391/commits/1dfe11998bcee3529ffc18332a00c385f4e14bb0

I'm not sure if there is a more optimised method for this.